src/Aquifer.API/Program.cs

```diff
@@ -23,8 +23,9 @@
     .AddOutputCache()
     .AddApplicationInsightsTelemetry()
     .AddSingleton<ITelemetryInitializer, RequestTelemetryInitializer>()
-    .AddDbContext<AquiferDbContext>(options =>
-        options.UseSqlServer(configuration?.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3)))
+    .AddDbContext<AquiferDbContext>(options => options
+        .UseSqlServer(configuration?.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
+        .EnableSensitiveDataLogging(sensitiveDataLoggingEnabled: builder.Environment.IsDevelopment()))
```

Comment:
Adding this line allows seeing parameters passed to SQL from EF in app logging but only in the Development environment.